after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
-  call gradle#setupGradleCommands()
-
   if android#isAndroidProject()
-    call android#setAndroidSdkTags()
-    call android#setClassPath()
-    call android#setupAndroidCommands()
+    autocmd BufEnter *.xml XMLns android
+    autocmd BufEnter AndroidManifest.xml XMLns manifest
```

Comment:
I think this autocmd better be in the plugin/gradle.vim as it independent of the file type and affects only one particular file. I never used XMLns before so I don't know if these commands add up or replace each other.